BUILDING.md

```diff
@@ -21,89 +19,72 @@ Start by understanding the layout of the source code:
 
 Note that currently only Windows, OSX, and Linux builds are officially supported.
 
-### Windows
+### Generating Bindings
 
-USD.NET.sln is a Visual Studio solution which includes all projects. The
-primary requirement is to setup the library and include paths for the
-C component of the build (UsdCs):
+A CMake build configuration is available for all platforms to build the bindings from the usd binaries. It's available through a build script in python
+and can build the native library (UsdCs), the C# library (USD.NET) and the tests.
 
- * Create a new environment variable USD_LOCATION pointing to your USD install root (contains /lib and /include)
+#### Requirements
 
+##### Cross Platforms
+ * Python 3.6 (available as python3 in your system PATH)
+ * USD 20.08 with python 3.6 support (used to generate type information for the SWIG bindings)
+ * USD 20.08 without python support` (used to minimize runtime dependencies of the bindings)
+ * CMake 3.19 (available in your system PATH)
+ * Swig 3.0.12 (available in your system PATH)
+ 
+ ##### Windows
+ * Visual Studio 2017
+ 
+ ##### Linux
+ * gcc 7
+ 
+ ##### OSX
+ * xcode v????
+ 
+#### Building USD 
 
-### OSX / Darwin
+We typically use the standard build steps from the USD repo instructions. The exact command line we are using are:
+* with python support: `python3 build_scripts/build_usd.py --build-monolithic --alembic --no-imaging --no-examples --no-tutorials ../artifacts/usd-v20.08`
+* no python support:   `python3 build_scripts/build_usd.py --build-monolithic --alembic --no-python --no-imaging --no-examples --no-tutorials ../artifacts/usd-v20.08_no_python`
 
-Similarly for OSX, an XCode project is provided in the src directory, called
-UsdCs.xcodeproj. The OSX build will produce a .bundle file, into which all
-dependent dylibs must be manually copied, as well as the USD plugInfo.json
-files. See the existing bundle as an example of the correct structure.
+**Unity developers**: USD binaries are built with each new release and pushed on Stevedore. See next section to download them automatically when building the bindings.
```

Comment:
Would be nice if these assets could be posted to the public stevedore so all external users could benefit from this goodness.

Reply:
I'll add that in the backlog. https://jira.unity3d.com/browse/USDU-156